dbt/include/trino/macros/catalog.sql

```diff
@@ -27,25 +27,35 @@
                     table_catalog as "table_database",
                     table_schema as "table_schema",
                     table_name as "table_name",
-                    null as "table_comment",
 
                     column_name as "column_name",
                     ordinal_position as "column_index",
                     data_type as "column_type",
-                    null as "column_comment"
+                    comment as "column_comment"
 
                 from {{ information_schema }}.columns
                 where
                     table_schema != 'information_schema'
                     and
                     table_schema in ('{{ schemas | join("','") | lower }}')
 
+            ),
+
+            table_comment as (
+
+                select
+                    catalog_name as "table_database",
+                    schema_name as "table_schema",
+                    table_name as "table_name",
+                    comment as "table_comment"
+
+                from system.metadata.table_comments
```

Comment:
Performance wise this is bad as it will retrieve comments on every connector that is configured. It would be better to retrieve only the catalogs and schemata that are actually used. Could we add a where clause here?

Reply:
Good point, I will add it

Reply:
Done